src/content/docs/alerts-applied-intelligence/applied-intelligence/anomaly-detection/anomaly-detection-applied-intelligence.mdx

```diff
@@ -12,33 +12,76 @@ redirects:
+import anomalydashboard from 'images/anomaly-dashboard.png'
 
-Notifications for anomalies can be delivered in Slack, or you can set up a webhook to deliver messages when you need them. These events are available for querying, creating custom dashboards, and alerting. After you set up a anomaly detection configuration (a group of apps you're interested in), you can add this configuration as a source. Then the anomalies will be automatically correlated with other data sources via incident intelligence.
+import anomalydetails from 'images/anomaly-details.png'
 
-To learn where anomalies appear, how you can use them to understand potential issues before they become incidents, and create alerts from them, watch this short video (approx. 4:15 minutes).
+import anomalyslacknotifications from 'images/anomaly-slack-notifications.png'
 
-<Video
-  id="Dhuo2GdR_7o"
-  type="youtube"
+import newRelicAlertsBaselineThresholds from 'images/new-relic-alerts-baseline-thresholds.png'
+
+import iconAlertCritical from 'images/icon-alert-critical.png'
+
+import iconAlertWarning from 'images/icon-alert-warning.png'
+
+import alertsHowCustomAnomaliesWork from 'images/alerts_screenshot-full_how-custom-anomalies-work.png'
+
+import alertsAnomaliesSetUpperandLowerRanges from 'images/alerts_screenshot-full_anomalies-set-upper-and-lower-ranges.png'
+
+import alertsFacetedAnomaliesTwo from 'images/alerts_screenshot-full_faceted-anomalies-two.png'
+
+import alertsFacetedAnomalies from 'images/alerts_screenshot-full_faceted-anomalies.png'
+
+import alertsConfigureYourAnomalyThresholds from 'images/alerts_screenshot-full_configure-your-anomaly-thresholds.png'
+
+With applied intelligence's anomaly detection, New Relic alerts your team of any unusual behavior instantly. New Relic uses AI to constantly observe your applications. We use this information to determine your application's baseline, or expected, performance. Whenever behavior deviates from the baseline, we know right away and alert your team so you can address any errors promptly and efficiently. 
+
+<img
+  title="A screenshot of the anomaly dashboard in the New Relic UI"
+  alt="A screenshot of the anomaly dashboard in the New Relic UI"
+  src={anomalydashboard}
 />
+<figcaption>The anomaly dashboard where your team can monitor any unusual behavior in your system.   
+  </figcaption>
 
-## Requirements
+## How we use anomalies [#anomalies-in-use]
 
-To use anomaly detection, ensure you have:
+At New Relic, our own developers know how important it is to monitor the health of our applications. We want our customers to access to the data they need whenever they need it so our team needs to be instantly alerted if there are any outliers in our system's performance. New Relic's anomaly detection uses applied intelligence to monitor three key golden signals: throughput, error rate and latency. With anomaly detection, our developers monitor the baseline performance for these metrics.
```

Comment:
At New Relic, our own developers know how important it is to monitor the health of our applications. We want our customers to have access to the data they need and be instantly alerted if there are outliers in their system's performance. New Relic uses applied intelligence to automatically monitor changes in golden signal data for your APM entities.